// @expo-google-fonts/cormorant-garamond/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const CormorantGaramond_300Light = require('./300Light/CormorantGaramond_300Light.ttf');
export const CormorantGaramond_400Regular = require('./400Regular/CormorantGaramond_400Regular.ttf');
export const CormorantGaramond_500Medium = require('./500Medium/CormorantGaramond_500Medium.ttf');
export const CormorantGaramond_600SemiBold = require('./600SemiBold/CormorantGaramond_600SemiBold.ttf');
export const CormorantGaramond_700Bold = require('./700Bold/CormorantGaramond_700Bold.ttf');
export const CormorantGaramond_300Light_Italic = require('./300Light_Italic/CormorantGaramond_300Light_Italic.ttf');
export const CormorantGaramond_400Regular_Italic = require('./400Regular_Italic/CormorantGaramond_400Regular_Italic.ttf');
export const CormorantGaramond_500Medium_Italic = require('./500Medium_Italic/CormorantGaramond_500Medium_Italic.ttf');
export const CormorantGaramond_600SemiBold_Italic = require('./600SemiBold_Italic/CormorantGaramond_600SemiBold_Italic.ttf');
export const CormorantGaramond_700Bold_Italic = require('./700Bold_Italic/CormorantGaramond_700Bold_Italic.ttf');
